### ISSUE TITLE
About hyperparameter settings

### ISSUE THREAD

**Issue by lovely-mao** [UkcheolShin/ThermalMonoDepth#5]
Hello
 your work is very inspiring.Thank you for sharing!

I found that the learning rate is set to 1e-6 in the paper, but the default parameter of the provided code is set to 1e-4. Similarly, the weight of the smoothing loss is set to 0.1 in the paper, but the default parameter of the provided code is set to 0.01. 

Which parameter should be used for better training results?

**Comment by UkcheolShin** [comment]
I remember that after publication, I found a more optimal hyperparameter to make the training process stable.
I think the currently provided code can provide more stable training results than the paper version.

**Comment by lovely-mao** [comment]
Thank you for your reply!
